docs/datamodels/source/devel/attributes.rst

```diff
@@ -61,26 +68,29 @@ is stored in a separate image extension).  Full-frame data for all other
 instruments have four columns or rows of reference pixels on each edge
 of the image.
 
-Model classes
--------------
-
-Base class
-''''''''''
+DataModel Base Class
+--------------------
 
 .. autoclass:: jwst.datamodels.DataModel
    :members:
 
-Concrete model classes
-''''''''''''''''''''''
+Specific Model Classes
+----------------------
```

Comment:
How is this list of model classes different than the one shown at the top of the doc ("current models")?